packages/pds-ember/blueprints/pds-component/files/__root__/components/__emberPrefix__/__name__/index.hbs

```diff
@@ -1,3 +1,10 @@
-<div class="<%= cssClassName %>" data-test-root>
+<div
+  class="<%= cssClassName %>"
+  ...attributes
+  data-test-root
```

Comment:
I still think you'll want unique data-keys per component here, but no reason to change it now.

Reply:
This is a good starting point and I've been changing this for each component anyway, but I didn't want to spend cycles getting it perfect.